src/mame/drivers/amstr_pc.cpp

```diff
@@ -845,7 +845,18 @@ ROM_START( pc3086 )
 	ROM_LOAD( "40178.ic801", 0x000, 0x800, CRC(f72f1c2e) SHA1(34897e78b3d10f96b36d81778e97c4a9a1b8618b) )
 ROM_END
 
-} // Anonymous namespace
+
+ROM_START( pc5086 ) // dies with error message 010
+	ROM_REGION16_LE( 0x20000, "bios", 0 )
+	ROM_LOAD( "c000.bin", 0x00000, 0x08000, CRC(5a8c640d) SHA1(7e5731f0febbad8228f758c6deceb550356c3b13) )
+	ROM_LOAD( "c800.bin", 0x08000, 0x02000, CRC(217ac584) SHA1(088aeb4bb389086c127274ddd3cde3048173cc8a) )
+	ROM_LOAD( "sys_rom.bin", 0x10000, 0x10000, CRC(d69b0d48) SHA1(3184d2a8107927631414bdcc4863e22b5a282def) )
+
+	ROM_REGION( 0x800, "keyboard", 0 ) // PC2086 / PC3086 102-key keyboard
+	ROM_LOAD( "40178.ic801", 0x000, 0x800, CRC(f72f1c2e) SHA1(34897e78b3d10f96b36d81778e97c4a9a1b8618b) )
+ROM_END
+
+}
```

Comment:
Why did you remove the comment closing the anonymous namespace?  It’s generally helpful for namespaces because the opening brace is quite distant.

Reply:
Hi Vas, that was a mistake on my side, sorry. 
I hadn't noticed the comment when I was last adding things, and thought I copypasted it to the end by mistake.